### PR TITLE
wip assign open close

### PR DESCRIPTION
**Pull request by josler**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -175,8 +175,17 @@ conversation.conversation_parts[1].body
...
+intercom.conversations.reply(:id => conversation.id, :type => 'admin', :admin_id => '123', :message_type => 'comment', :body => 'bar')
+
+# Open
+intercom.conversations.open(id: conversation.id, admin_id: '123')
+
+# Close
+intercom.conversations.close(id: conversation.id, admin_id: '123')
+
+# Assign
+intercom.conversations.assign(id: conversation.id, admin_id: '123', assignee_id: '124')
```
Comment (dehora) [inline README.md]: worth showing an example of reply with close/open?

Comment (bobjflong) [inline README.md]: i think we made a call to leave these out of ruby client (and maybe go) for now. cc @josler

Comment (bobjflong) [inline README.md]: by leave out, I guess I mean "document". Because it is possible using `#reply`

Comment (josler) [inline README.md]: yeah, just a doc example wold be enough

Comment (bobjflong) [inline README.md]: done below

**Comment by josler** [comment]
:sagittarius:

**Pull request merged**